features/openhab-addons-external/src/main/resources/conf/gcal.cfg

```diff
@@ -0,0 +1,10 @@
+# Credentials for Google API and Google Calendar name
+
+# Copied from Credentials page on console.developers.google.com
+#client_id=
+
+# Copied from Credentials page on console.developers.google.com
+#client_secret=
+
+# The name of the Google Calendar that GCal should use
+#calendar_name=
```

Comment:
Sure. Good point. I didn't want to repeat what was in the Wiki, but I certainly can add more to the comments.

As for openhab_default.cfg, I'm not sure what you're asking as I didn't change that file.  It still contains the gcal: prefixes.